tests/returntypes/test_parameter_shift_shot_vec_new.py

```diff
@@ -639,11 +649,10 @@ def test_pauli_rotation_gradient(self, mocker, G, theta, shift, tol):
 
         assert spy.call_args[1]["shifts"] == (shift,)
 
-        # TODO: finite diff shot-vector update
-        # tapes, fn = qml.gradients.finite_diff(tape)
-        # numeric_val = fn(dev.batch_execute(tapes))
-        # for a_val, n_val in zip(autograd_val, numeric_val):
-        #     assert np.allclose(a_val, n_val, atol=shot_vec_tol, rtol=0)
+        tapes, fn = qml.gradients.finite_diff(tape, h=10e-2, shots=shot_vec)
```

Comment:
`10e-2` in Python is just `0.1`. I don't know if you intended `0.1` or `1e-2` here, but using either of those instead would be clearer for future people looking at this. Doing a simple find-and-replace should be sufficient.

Reply:
Sure. :+1: